### PR TITLE
Run kubevirt lane with KUBEVIRT_PSA=true

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:
To be able to run VMs on a namespace with restricted PSA policy, KubevirtSeccompProfile feature gate should be enabled and a new configuration needs to be enabled under
.spec.configuration.seccompConfiguration

```
virtualMachineInstanceProfile:
   customProfile:
       localhostProfile: kubevirt/kubevirt.json
```

Reference: https://github.com/kubevirt/kubevirt/pull/8917

**Special notes for your reviewer**:

### CODE REVIEW

File: manifests/secondarydns.yaml [hunk 1]
```diff
@@ -143,3 +143,18 @@ spec:
           name: secondary-dns
       - name: secdns-zones
         emptyDir: {}
+---
+apiVersion: kubevirt.io/v1
```
Comment (oshoval) [inline manifests/secondarydns.yaml:147]: Is there a risk (or even just redundant) that it is already created when 
deployed as part of CNAO / HCO (or D/S) ?
If so, we will need to create it only where needed

Comment (AlonaKaplan) [reply manifests/secondarydns.yaml:147]: I will add it as part of `cluster-sync`. If ksd is deployed by CNAO/HCO then let's them to decide whether the fg is enabled or not.

File: cluster/sync.sh [hunk 2]
```diff
@@ -52,3 +52,4 @@ pods_ready_wait() {
 
 pods_ready_wait
 make create-nodeport
+./cluster/kubectl.sh apply -f ./hack/psa/kubevirt.yaml
```
Comment (oshoval) [inline cluster/sync.sh:55]: please create a function for it, like pods_ready_wait

**Review by oshoval — COMMENTED** [review]
The previous run failed, 
are the latest changes fixed it, or it can be a flake ?

Thanks

File: cluster/up.sh [hunk 3]
```diff
@@ -18,7 +18,7 @@ set -ex pipefail
 
 export DEPLOY_CNAO=${DEPLOY_CNAO:-true}
 export DEPLOY_KUBEVIRT=${DEPLOY_KUBEVIRT:-true}
-export KUBEVIRT_PSA=${KUBEVIRT_PSA:-false}
+export KUBEVIRT_PSA=${KUBEVIRT_PSA:-true}
```
Comment (oshoval) [inline cluster/up.sh:21]: It deploys kubevirt v0.59.0 (line 26)
does it support PSA already ?
(well i guess otherwise it would fail isn't it ?)

**Comment by AlonaKaplan** [comment]
> The previous run failed, are the latest changes fixed it, or it can be a flake ?
> 
> Thanks

It failed to communicate with the webhook. IDK why it happened.

File: cluster/sync.sh (hunk 2 above)
Comment (AlonaKaplan) [reply cluster/sync.sh:55]: done.

**Comment by oshoval** [comment]
> > The previous run failed, are the latest changes fixed it, or it can be a flake ?
> > Thanks
> 
> It failed to communicate with the webhook. IDK why it happened.

sounds like unrelated flake, saw in the past here and there, unless we see it more frequent

**Review by oshoval — APPROVED** [review]

**Pull request merged**